src/locales/en_US/translation.json

```diff
@@ -4,6 +4,13 @@
   "header": {
     "connectWallet": "CONNECT WALLET"
   },
+  "toast": {
+    "walletConnected": "Wallet connected",
+    "walletDisconnected": "Wallet disconnected",
+    "transactionStarted": "Transaction started...",
+    "transactionMined": "Transaction processing...",
+    "transactionFinished": "Transaction succeeded..."
```

Comment:
"finished" mark the end of an event, not an ongoing one, so this line could be

```
    "transactionFinished": "Transaction succeeded"
```

Reply:
yes, already fixed here

Reply:
doesn't look fixed when I CTRL+SHIFT+F5 refresh, but I'll take your word for it

Reply:
![image](https://user-images.githubusercontent.com/22704999/107833733-80e4d580-6d94-11eb-9a8d-6d0cb41187b7.png)

Reply:
huh, github isn't updating for the second commit, it shows up [here](https://github.com/wolvesofwallstreet/wolves.finance/commit/94b3cc570a0bc8a851968e901548814567b3e2b7#diff-1d05998577b8b78bb4c40767b2b6bb40eccebc023376eb214a987788a7ce6ec6R12) but not in the files changed